R/Overview_Table.R

```diff
@@ -69,7 +68,7 @@ Overview_Table <- function(
       ) %>%
       left_join(
         dfSite %>%
-          mutate(site_num = as.character(.data$site_num)) %>%
+          mutate(site_num = as.character(.data[[Read_Mapping()$dfSITE$strSiteCol]])) %>%
```

Comment:
@andersonjake1988, @mattroumaya - I don't think this is going to work with non-standard column mappings.

Reply:
well shoot... could you expand on non-standard column mapping? as far as what to expect, so I can modify this?

Reply:
Sure thing!  The input data to `{gsm}` might not have a column in `dfSITE` named `site_num`.  In this scenario the user/study team would modify the `dfSITE` column mapping as needed (`dfSITE$strSiteCol`).

Rather than pass the custom column mapping into `Overview_Table` we can just assume the mapped site data, the output of `Site_Map_Raw()`, contains a column named `siteid` ([pertinent code](https://github.com/Gilead-BioStats/gsm/blob/dev/R/Site_Map_Raw.R#L19-L21)).  `Site_Map_Raw()` adds a column named `siteid` by setting it to `dfSITE[[ mapping$dfSITE$strSiteCol ]]`.

Reply:
And to elaborate a bit more, we either want to work with raw data + mappings or with mapped data, so we're not making assumptions about the columns in the data.